pkgs/by-name/wa/wahay/package.nix

```diff
@@ -0,0 +1,60 @@
+{
+  lib,
+  stdenv,
+  buildGoModule,
+  fetchFromGitHub,
+  gnumake,
+  pkg-config,
+  gtk3,
+  git,
+  rubyPackages,
+}:
+
+buildGoModule rec {
+  pname = "wahay";
+  version = "0.2";
+
+  src = fetchFromGitHub {
+    owner = "digitalautonomy";
+    repo = "wahay";
+    rev = "v${version}";
+    hash = "sha256-59gVGDElIs+OQ/ELwOFjz6YqG18iHqoa2yYF2Ddi0/o=";
+  };
+
+  vendorHash = "sha256-w8lUnvy5dPMHWbzyyTq9Q/kE/4vSuOHffaY9CeasvQ0=";
+
+  buildInputs = [
+    gtk3
+  ];
+
+  nativeBuildInputs = [
+    pkg-config
+    gnumake
+    git
+    rubyPackages.sass
+  ];
+
+  #
+  buildPhase = ''
+    runHook preBuild
+
+    make BUILD_DIR=$out/bin TAG_VERSION=v${version} build
```

Comment:
I’d suggest to override `BUILD_TIMESTAMP` to enable reproducible builds:

`BUILD_TIMESTAMP="$(date -u -d "@$SOURCE_DATE_EPOCH" '+%Y-%m-%d %H:%M:%S')"`